rosidl_generator_py/resource/_msg.py.em

```diff
@@ -334,7 +334,18 @@ if isinstance(type_, AbstractNestedType):
         typename = self.__class__.__module__.split('.')
         typename.pop()
         typename.append(self.__class__.__name__)
+@[if 'import array' in imports]@
+        args = []
+        for s in self.__slots__:
+            field = getattr(self, s, None)
+            if type(field) == array.array:
+                t = repr(field.tolist())
```

Comment:
Since this operation is fairly expensive (in terms of memory usage) I would suggest to keep using `repr(field)` and then process the string to remove the `array(typecode, )` part.

Reply:
I did this in 079d988.